protostar-rust/tests/data/declare_test/Scarb.toml

```diff
@@ -8,3 +8,4 @@ sierra = true
 [dependencies]
 starknet = ">=1.1.0"
 
+[tool.protostar]
```

Comment:
this should not be required. I assume you added this to avoid errors but we should write code that allows this section to be missing.

Reply:
There's an issue for that #2041